tests/versioning_3_test.go

```diff
@@ -1968,7 +1969,7 @@ func (s *Versioning3Suite) setCurrentDeployment(tv *testvars.TestVars) {
 		}
 		_, err := s.FrontendClient().SetWorkerDeploymentCurrentVersion(ctx, req)
 		var notFound *serviceerror.NotFound
-		if errors.As(err, &notFound) {
+		if errors.As(err, &notFound) || (err != nil && err.Error() == workerdeployment.ErrCurrentVersionDoesNotHaveAllTaskQueues) {
```

Comment:
```suggestion
		if errors.As(err, &notFound) || errors.Is(err, serviceerror.NewFailedPrecondition(workerdeployment.ErrCurrentVersionDoesNotHaveAllTaskQueues)) {
			return false
		}
```

nit: maybe this looks a little better given that this is a failedPrecondition error

Reply:
thanks!